### PR TITLE
[GEOS-7086] Make sure if namespace is deleted, associated workspace is deleted too

### PR DESCRIPTION
**Pull request by ridethepenguin**
See mailing list and associated [JIRA issue](https://osgeo-org.atlassian.net/browse/GEOS-7086) for discussion.

### CODE REVIEW

File: src/main/src/main/java/org/geoserver/catalog/NamespaceWorkspaceConsistencyListener.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2015 Open Source Geospatial Foundation - all rights reserved
```
Comment (mbarto) [inline src/main/src/main/java/org/geoserver/catalog/NamespaceWorkspaceConsistencyListener.java]: You should add 2015 without removing 2014, 2014 - 2015

File: src/restconfig/src/test/java/org/geoserver/catalog/rest/NamespaceTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2015 Open Source Geospatial Foundation - all rights reserved
```
Comment (mbarto) [inline src/restconfig/src/test/java/org/geoserver/catalog/rest/NamespaceTest.java]: Same as above

**Comment by ridethepenguin** [comment]
Thanks @mbarto, headers fixed.

**Comment by simboss** [comment]
Is there a JIRA issue for this PR? Are we going to backport?

**Comment by ridethepenguin** [comment]
@simboss created issue [GEOS-7086](https://osgeo-org.atlassian.net/browse/GEOS-7086) on JIRA. Backporting should be straightforward, I'll do it after this is merged.

**Comment by aaime** [comment]
Merged, thanks

**Pull request merged**